tests/conftest.py

```diff
@@ -41,6 +43,21 @@
 )
 
 
+TMP_DIR = "/tmp/pytest_tmp_path/"
```

Comment:
This seems to be used only in the `tmp_path` fixture bellow, so could be made local to that function, no need to stay in the module scope.